src/cluster.cpp

```diff
@@ -49,10 +49,15 @@ getClusterReader(const Reader& zimReader, offset_t offset, CompressionType* comp
 {
   uint8_t clusterInfo = zimReader.read(offset);
   // Very old zim files used 0 as a "default" compression, which means no compression.
-  if(clusterInfo&0x0F == 0) {
+  uint8_t compInfo = clusterInfo&0x0F;
```

Comment:
A space before and after the & operator can improve readability

Reply:
Done